src/components/security-analysis-result.js

```diff
@@ -412,57 +419,57 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
     const nmKConstraintsColumns = useMemo(() => {
         return [
             {
-                width: 200,
+                flex: 1,
```

Comment:
you can add flex: 1 only on columnDefs
for both renderTableNmKConstraints and renderTableNmKContingencies (then, you can remove width for columnsNmKContingencies)